docs/source/index.rst

```diff
@@ -13,13 +13,22 @@ information, see the `Snowpark Developer Guide for Python <https://docs.snowflak
 .. toctree::
    :maxdepth: 2
 
-   snowpark_classes
-   functions
+   session
+   IO
+   dataframe
+   column
+   types
+   row
+   functions 
+   window 
+   grouping 
    table_function
+   table
    stored_procedures
-   types
    udf
    udtf
    exceptions
+   observability
+   async_job
```

Comment:
should we do them in sorted order?

Reply:
I am roughly following PySpark's order here. But I'm also open to using sorted order.

Reply:
Good question. I was also thinking about ordering. How about moving `async_job` next after `table` and `observability` just before `exceptions`?

Reply:
@sfc-gh-yixie , sounds good